utils/docker/Dockerfile.centos.7

```diff
@@ -123,17 +123,18 @@ enabled=1\n\
 gpgcheck=False\n" >> /etc/yum.repos.d/$repo:$branch:$build_number.repo;          \
         done;                                                                    \
         pkgs="cart-devel$CART_COMMIT argobots-devel libpmem-devel                \
-              libpmemobj-devel fuse-devel protobuf-c-devel spdk-devel            \
-              fio-3.3-4.el7 libisa-l-devel mercury-devel-1.0.1-13.el7            \
+              libpmemobj-devel fuse-devel protobuf-c-devel                       \
+              spdk spdk-devel                                                    \
+              fio-3.3-5.el7 libisa-l-devel mercury-devel-1.0.1-19.el7            \
               openpa-devel libfabric-devel                                       \
-              ompi-devel pmix-devel hwloc-devel dpdk-devel-18.02";               \
+              ompi-devel pmix-devel hwloc-devel dpdk-devel";                     \
```

Comment:
This is changing something in the build.  Is it safe to do so without running tests?

Reply:
Tests were run for that change.  It was needed to get them to pass.
This is used for the "quickbuild" where only DAOS is built and RPMs are used for the components.
This is needed for Coverity to only instrument a scan of DAOS.

Reply:
This is why for now, I have added a Coverity scan for all PRs instead of just the master branch, to make sure that a PR does not break "quickbuild", which previously was just used for speeding up some test PRs.